lawnchair/src/app/lawnchair/ui/preferences/components/IconShapePreference.kt

```diff
@@ -72,8 +97,10 @@ fun iconShapeEntries(context: Context): List<ListPreferenceEntry<IconShape>> {
 fun IconShapePreference(
 ) {
     val context = LocalContext.current
+    val preferenceManager2 = preferenceManager2()
     val entries = remember { iconShapeEntries(context) }
-    val adapter = preferenceManager2().iconShape.getAdapter()
+    val iconShapeAdapter = preferenceManager2.iconShape.getAdapter()
+    val customIconShape = preferenceManager2.customIconShape.asState()
 
     PreferenceLayout(label = stringResource(id = R.string.icon_shape_label)) {
         PreferenceGroup {
```

Comment:
Maybe add a label `Presets` in this part:

```suggestion
        PreferenceGroup (
            heading = stringResource(id = R.string.presets),
        ) {
```

Reply:
https://github.com/LawnchairLauncher/lawnchair/pull/3016#discussion_r993416044